server/router/index.js

```diff
@@ -1,13 +1,13 @@
 
 const express = require('express');
-const { logout, login } = require('../controllers');
-
 const { auth } = require('../controllers/middlewares');
+const { logout, login, teacherProfile } = require('../controllers');
 
 const router = express.Router();
 
 router.post('/login', login);
 router.use(auth);
+router.get('/profile/teacher', teacherProfile);
```

Comment:
and here, again if this is supposed to search for a specific profile, you'd need
```
router.get('/profiles/teachers/:id', teacherProfile);
```